### PR TITLE
Ported fab commands for UI devs For those that don't want to deal with setting up python and fabric locally

### PR DESCRIPTION
**Pull request by gridcell**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,34 @@
 
-.DEFAULT:
-	fab "$@"
\ No newline at end of file
+down:
+	@docker-compose down
+
+build_image:
```
Comment (saanobhaai) [inline Makefile:5]: Maybe we should use same syntax as in fabric to decrease potential future confusion? i.e. `buildnocache` for this one
Also, don't we need to execute `./ci_cd/version.sh` here?

File: Makefile [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+	@docker-compose build --no-cache --pull
+
+up:
+	@docker-compose up -d
+
+dbrestore:
+	@docker exec -it api_service python manage.py dbrestore local
+
+migrate:
+	@docker exec -it api_service python manage.py migrate
```
Comment (saanobhaai) [inline Makefile:15]: Don't know if it's worth making `api_service` a variable...

File: Makefile [hunk 3]
```diff
@@ -1,3 +1,34 @@
...
+down:
+	@docker-compose down
+
+build_image:
+	@docker-compose build --no-cache --pull
+
+up:
+	@docker-compose up -d
+
+dbrestore:
```
Comment (saanobhaai) [inline Makefile:11]: wouldn't hurt to do `dbbackup` too maybe?

File: Makefile [hunk 4]
```diff
@@ -1,3 +1,34 @@
...
+	@docker exec -it api_service python manage.py dbrestore local
+
+migrate:
+	@docker exec -it api_service python manage.py migrate
+
+fresh_install:
```
Comment (saanobhaai) [inline Makefile:17]: as above -- maybe `freshinstall`

**Comment by mmnoo** [comment]
Could these changes come with an update to the readme?

**Pull request merged**